validations.md

```diff
@@ -0,0 +1,102 @@
+# Detailed Validation Breakdown
```

Comment:
I would rename the title to just `#Validations` or something like that and add a short description of what this document is